code/game/objects/items/weapons/implants/types/explosive_implant.dm

```diff
@@ -43,16 +43,17 @@
 
 		qdel(src)
 
-/obj/item/weapon/implant/explosive/implanted(mob/source)
+/obj/item/weapon/implant/explosive/implanted(mob/source, mob/target)
 	if(malfunction == IMPLANT_MALFUNCTION_PERMANENT)
 		return 0
 	phrase = input("Choose activation phrase:") as text
 	var/static/list/replacechars = list("'" = "", "\"" = "", ">" = "", "<" = "", "(" = "", ")" = "")
 	phrase = sanitize_simple(phrase, replacechars)
-	usr.mind.store_memory("Explosive implant in [source] can be activated by saying something containing the phrase ''[src.phrase]'', <B>say [src.phrase]</B> to attempt to activate.", 0, 0)
-	to_chat(usr, "The implanted explosive implant in [source] can be activated by saying something containing the phrase ''[src.phrase]'', <B>say [src.phrase]</B> to attempt to activate.")
+	usr.mind.store_memory("Explosive implant in [target] can be activated by saying something containing the phrase ''[src.phrase]'', <B>say [src.phrase]</B> to attempt to activate.", 0, 0)
```

Comment:
I made sure `imp_in` is always set to the mob that's been implanted when `implanted` is called. You can do away with this new `mob/target` parameter and use `imp_in` instead. It would be consistent with the other `implanted` implementations

Reply:
👍